src/main/java/io/jenkins/plugins/coverage/CoveragePublisher.java

```diff
@@ -54,6 +54,8 @@
     private boolean calculateDiffForChangeRequests = false;
     private boolean failBuildIfCoverageDecreasedInChangeRequest = false;
 
+    private boolean publishChecks = false;
```

Comment:
Don’t we want to default to publishing? Change to skipPublishingChecks?

Reply:
either is ok, I'll change it